src/azure-cli/azure/cli/command_modules/sql/aaz/latest/sql/mi/__cmd_group.py

```diff
@@ -0,0 +1,23 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command_group(
+    "sql mi",
+)
+class __CMDGroup(AAZCommandGroup):
+    """Managed Instance
```

Comment:
Will this overwrite the original command group help for `az sql mi`?

Reply:
looks like it does clash with the original help definition, I'll fix this and the models
cc: @kairu-ms